<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NFCRFID</class>
 <widget class="QWidget" name="NFCRFID">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="main_layout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="combo_box_tag_type">
       <item>
        <property name="text">
         <string>Mifare Classic</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NFC Forum Type 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NFC Forum Type 2</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_scan">
       <property name="text">
        <string>Scan for Tag</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_id">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_read_page">
       <property name="text">
        <string>Start Page:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinbox_read_page">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_read_key">
       <property name="text">
        <string>Key:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combobox_read_key">
       <item>
        <property name="text">
         <string>A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>B</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_read_spinbox" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_5"/>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QPushButton" name="button_read_page">
       <property name="text">
        <string>Read Page</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPlainTextEdit" name="textedit_read_page">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
       <property name="undoRedoEnabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="button_write_page">
       <property name="text">
        <string>Write Page</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="write_page_grid">
     <item row="0" column="0">
      <widget class="QLabel" name="write_page_label1">
       <property name="text">
        <string>Byte 0-3:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="layout_write2"/>
     </item>
     <item row="1" column="3">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="layout_write3"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="write_page_label4">
       <property name="text">
        <string>Byte 12-15:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="write_page_label2">
       <property name="text">
        <string>Byte 4-7:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="write_page_label3">
       <property name="text">
        <string>Byte 8-11:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="layout_write1"/>
     </item>
     <item row="2" column="3">
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="layout_write4"/>
     </item>
     <item row="3" column="3">
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
